specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyMetadata.json

```diff
@@ -260,6 +260,9 @@
           "items": {
             "$ref": "#/definitions/ErrorDefinition"
           },
+          "x-ms-identifiers": [
+            "code"
```

Comment:
>code [](http://example.com/codeflow?start=13&length=4)

there could be duplicates in here when multiple errors occur.  There isn't a unique combination of properties